<launch>
<!-- Launch file for Livox AVIA LiDAR -->

	<arg name="rviz" default="true" />

	<rosparam command="load" file="$(find ww_lio)/config/avia.yaml" />

    <node pkg="ww_lio" type="ww_lio_map" name="laserMapping" output="screen" /> 

	<group if="$(arg rviz)">
	<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find ww_lio)/rviz_cfg/loam_livox.rviz" />
	</group>

</launch>
